### PR TITLE
Addresses #68

### PR DESCRIPTION
**Pull request by palanik1**
Addresses #68

### CODE REVIEW

File: asset/README.MD [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Annotations of code files for <project>
+ - **Repository:** <project git url>
+ - **Copyrights:** 
+    - The source code copyrights rest with original copyright holders as mentioned in above repository link
+    - The copyright for comments rest with comment authors
```
Comment (dushyantbehl) [inline asset/README.MD:5]: Along with copyright should we say this line? or something like this.

`The comments are thoughts of the author and not of maintianers of this repo.`

File: scripts/execute_workflow.sh [hunk 2]
```diff
@@ -14,12 +14,14 @@ INTERMEDIATE_FILES=${PROJECT_DIR}/intermediate_files
...
 ANNOTATOR=src/annotator.py
 COMMENT_EXTRACTOR=src/utils/comment_extractor.py
+README=asset/README.MD
 
 function create_dirs {
     # Create the directories
     mkdir -p ${PROJECT_DIR}
     mkdir -p ${ORIGINAL_SRC} ${HUMAN_ANNOTATED} ${INTERMEDIATE_FILES}
-
+    cp asset/README.MD ${PROJECT_DIR}
```
Comment (dushyantbehl) [inline scripts/execute_workflow.sh:23]: did you meant to write `cp ${README} ${PROJECT_DIR}`

**Review by dushyantbehl — CHANGES_REQUESTED** [review]

File: asset/README.MD (hunk 1 above)
Comment (palanik1) [reply asset/README.MD:5]: I think we will have a higher level README in the top of the project saying maintainers dont take any responsibility for the comments/veracity of information. Good point!

File: scripts/execute_workflow.sh (hunk 2 above)
Comment (palanik1) [reply scripts/execute_workflow.sh]: fixed

File: asset/README.MD (hunk 1 above)
Comment (palanik1) [reply asset/README.MD:5]: I would prefer adding it top level.

Comment (dushyantbehl) [reply asset/README.MD:5]: sure

**Review by dushyantbehl — APPROVED** [review]
Looks good

**Pull request merged**